test/test_cmake/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Mike Dev
+# Distributed under the Boost Software License, Version 1.0.
+# See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt
+#
+# NOTE: This does NOT run the unit tests for Boost.Mpl.
+#       It only tests, if the CMakeLists.txt file in it's root works as expected
+
+cmake_minimum_required( VERSION 3.5 )
+
+project( BoostBindCMakeSelfTest )
+
+add_subdirectory( ../../../assert ${CMAKE_CURRENT_BINARY_DIR}/libs/assert )
```

Comment:
Static list of dependencies is going to cause long-term maintenance problems.

Reply:
Two reasonable alternatives I see: 

- Just remove the test altogether- that is what peter dimov argues for
- Instead of listing them explicitly,  grep for all sibling projects that have a cmake file and assume only the necessary repositories are checked out.